java/server/src/org/openqa/selenium/grid/log/LoggingOptions.java

```diff
@@ -171,4 +238,13 @@ private OutputStream getOutputStream() {
         })
         .orElse(System.out);
   }
+
+  private String getJSONString(Map<String, Object> map) {
```

Comment:
`Json`, not `JSON`. We try and avoid shouting on this project :)

Reply:
Will keep in mind. :)